gbfs.md

```diff
@@ -369,6 +378,7 @@ Field Name | REQUIRED | Type | Defines
     "phone_number": "1-800-555-1234",
     "name": "Example Bike Rental",
     "operator": "Example Sharing, Inc",
+    "opening_hours": "Apr 1 - Nov 3 00:00-24:00"
```

Comment:
Sorry, the missing space in this line must have distracted me from the missing comma at the end yesterday
```suggestion
    "opening_hours": "Apr 1 - Nov 3 00:00-24:00",
```